### ISSUE TITLE
Leaflet map shows error for "full screen" functionality

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3477]
https://publiclab.org/puerto-rico

```
Uncaught TypeError: L.Control.Fullscreen is not a constructor
    at setupFullScreen
```

this shows up in the JavaScript console! I think maybe @sagarpreet-chadha has a suggestion on this?

**Comment by jywarren** [comment]
There should be a code file in `/app/assets/javascripts/...` named leaflet something!

**Comment by sagarpreet-chadha** [comment]
Hi @jywarren ...should i solve this one and other related Maps issue OR make new detailed issues ?

**Comment by jywarren** [comment]
Either way... however you'd like to help!!!

**Comment by sagarpreet-chadha** [comment]
This error is removed . The full screen mode is available at /people , /profile and /maps 😄 !

**Comment by sagarpreet-chadha** [comment]
Solved in #4544 !
